src/VersionChip/VersionChip.test.tsx

```diff
@@ -35,4 +37,51 @@ describe("VersionChip", () => {
     render(<VersionChip version="1.1"></VersionChip>);
     expect(screen.getByTestId("AccountTreeIcon")).toBeInTheDocument();
   });
+
+  // check background color when the `selected` prop is true
+  it("applies correct background color when `selected` is true", () => {
+    render(<VersionChip version="1.0" selected={true} />);
+    const chipElement = screen.getByTestId("version-chip");
+    const styles = window.getComputedStyle(chipElement);
+    // passing resolved color from alpha(theme.palette.info.main, 0.12) and theme.palette.primary.main
+    expect(styles.backgroundColor).toBe("rgba(2, 136, 209, 0.12)");
+    expect(styles.border).toBe("1px solid #1976d2");
```

Comment:
Please be consistent with the color encoding, you've used rgba mostly, lets stick to that here too
```suggestion
    expect(styles.border).toBe("1px solid rgba(25, 118, 210, 1)");
```

Reply:
I tried that before, however we'll get this error, as we receiving the style in hex format
![image](https://github.com/user-attachments/assets/49dc453c-d6a0-44c6-a43d-53fdd8fecdbc)

Reply:
In this case proceed with the hex value